pyensembl/database.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, gtf, auto_download=False):
     def __eq__(self, other):
         return isinstance(other, Database) and self.gtf == other.gtf
 
+    def __hash__(self):
```

Comment:
Sorry that this is unrelated to this PR, but can we add to class docstring (or somewhere else) what `gtf` is? Or perhaps rename that to not be an abbreviation